### DISCUSSION TITLE
Do Not Display Custom Image Over Map for Hybrid Meetings

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#922] [Q&A] [answered]
Referencing [https://github.com/code4recovery/tsml-ui/issues/109](url), is it possible to display the map if it's a hybrid meeting and not a custom graphic in TSML-ui?

I realize this is a front-end cosmetic issue and is not a bug in the back end. But perhaps it's a quick answer? 

Thanks

**Comment by joshreisner** [comment, marked as answer]
try adding the `:not` selector like this:

```
#tsml-ui .meeting.online:not(.in-person) .map::after {
  background-image: url(https://images.unsplash.com/photo-1588196749597-9ff075ee6b5b?crop=entropy&cs=tinysrgb&fit=crop&fm=jpg&h=1440&ixid=MnwxfDB8MXxhbGx8fHx8fHx8fHwxNjIyMTIzODkw&ixlib=rb-1.2.1&q=80&utm_campaign=api-credit&utm_medium=referral&utm_source=unsplash_source&w=1920);
  background-position: center center;
  background-repeat: no-repeat;
  background-size: cover;
  bottom: 0;
  content: '';
  left: 0;
  opacity: 0.25;
  position: absolute;
  right: 0;
  top: 0;
}
```

**Reply by Grateful-Viv** [reply]
Thanks, @joshreisner

That was what I needed `#tsml-ui .meeting.online:not(.in-person) .map::after { `instead of `#tsml-ui .meeting.online .map::after {`

**Reply by joshreisner** [reply]
ha i just noticed the image you're using with the intergroup logo on it, nicely done!

**Reply by Grateful-Viv** [reply]
Thanks, Josh. 😏
